app/views/ReportView/components/Expression/index.jsx

```diff
@@ -182,7 +182,7 @@ const Expression = () => {
               ))}
             </Paper>
           )}
-          {comparators && !Boolean(comparators.length) && (
+          {comparators && !comparators.length && (
```

Comment:
unrelated to this line

on line 34 I believe it should be initialized to  `const [expOutliers, setExpOutliers] = useState([])`;

LinearProgress stays when expOutliers stay `undefined`

Reply:
Line 62 should handle this case. I won't change this here if it's working since it's unrelated to the pr

Reply:
ah, it seems like this was due to the API problem 

```
Uncaught (in promise) TypeError: images.find is not a function
```